src/runtime/test/manifest-test.ts

```diff
@@ -644,45 +644,46 @@ ${particleStr1}
-          ${args.isRequiredSlotA ? 'must ' : ''}consume slotA
-          ${args.isRequiredSlotB ? 'must ' : ''}consume slotB
+          ${isRequiredSlotA ? 'must ' : ''}consume slotA
+          ${isRequiredSlotB ? 'must ' : ''}consume slotB
 
         recipe
           slot 'slota-0' as s0
           SomeParticle
             consume slotA as s0
       `)).recipes[0];
       recipe.normalize();
-      assert.equal(args.expectedIsResolved, recipe.isResolved());
+      assert.equal(recipe.isResolved(), expectedIsResolved);
     };
-    await parseRecipe({isRequiredSlotA: false, isRequiredSlotB: false, expectedIsResolved: true});
-    await parseRecipe({isRequiredSlotA: true, isRequiredSlotB: false, expectedIsResolved: true});
-    await parseRecipe({isRequiredSlotA: false, isRequiredSlotB: true, expectedIsResolved: false});
-    await parseRecipe({isRequiredSlotA: true, isRequiredSlotB: true, expectedIsResolved: false});
+    await parseRecipe(false, false, true);
+    await parseRecipe(true, false, true);
+    await parseRecipe(false, true, false);
+    await parseRecipe(true, true, false);
   });
   it('SLANDLES multiple consumed slots', async () => {
-    const parseRecipe = async (args) => {
+    const parseRecipe = async (label: string, isRequiredSlotA: boolean, isRequiredSlotB: boolean, expectedIsResolved: boolean) => {
       const recipe = (await Manifest.parse(`
         particle SomeParticle in 'some-particle.js'
-          \`consume${args.isRequiredSlotA ? '' : '?'} Slot slotA
-          \`consume${args.isRequiredSlotB ? '' : '?'} Slot slotB
+          \`consume${isRequiredSlotA ? '' : '?'} Slot slotA
+          \`consume${isRequiredSlotB ? '' : '?'} Slot slotB
 
         recipe
           \`slot 'slota-0' as s0
           SomeParticle
             slotA consume s0
       `)).recipes[0];
-      recipe.normalize();
-      assert.equal(args.expectedIsResolved, recipe.isResolved(),
-                   `Expected recipe to be ${args.expectedIsResolved ? '' : 'un'}resolved`);
+      const options = {errors: new Map(), details: '', showUnresolved: true};
+      recipe.normalize(options);
+      console.log(recipe.obligations);
+      assert.equal(recipe.isResolved(options), expectedIsResolved, `${label}: Expected recipe to be ${expectedIsResolved ? '' : 'un'}resolved.\nErrors: ${JSON.stringify([...options.errors, options.details])}`);
     };
-    await parseRecipe({isRequiredSlotA: false, isRequiredSlotB: false, expectedIsResolved: true});
-    await parseRecipe({isRequiredSlotA: true, isRequiredSlotB: false, expectedIsResolved: true});
-    await parseRecipe({isRequiredSlotA: false, isRequiredSlotB: true, expectedIsResolved: false});
-    await parseRecipe({isRequiredSlotA: true, isRequiredSlotB: true, expectedIsResolved: false});
+    await parseRecipe('1', false, false, true);
+    await parseRecipe('2', true, false, true);
+    await parseRecipe('3', false, true, false);
+    await parseRecipe('4', true, true, false);
```

Comment:
I think the idea of the args object was that it was documenting the meaning of the boolean flags.

Reply:
I think things have improved thanks to our use of typescript.
I'll put it back for now (and use it in the non-slandles test as well) until I feel more strongly about it, as I'm currently unsure that the conciseness is worth the potential cost to readability .

Reply:
Interesting point re: typescript. Maybe it's worth following up with Maria (she wrote these tests)?